test/Microsoft.Health.Fhir.R4.Tests.E2E/testconfiguration.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+    "FhirServer": {
+        "Cors": {
+            "Origins": [ "https://localhost:6001" ],
+            "Methods": [ "*" ],
+            "Headers": [ "*" ],
+            "MaxAge": 1440
+        },
+        "Operations": {
+            "Terminology": {
+                "ExternalTerminologyServer": "http://tx.fhir.org/r4",
+                "Validate": {
+                    "CacheDurationInSeconds": 14400,
+                    "ProfileValidationTerminologyServer": "http://hapi.fhir.org/baseR4"
```

Comment:
One of the issues we had with this branch was the test severs not being reliable for running E2E tests. We wanted to replace them with our in house server, but that didn't get done in time. 
We could skip all the E2E tests for terminology, but I don't feel great about including a feature that doesn't have tests run against it.

Reply:
This could be done with an in-memory terminology server making it less dependent on external systems.